recipes/pydrift/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pydrift" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 572a5f5ee3cfde052ee858cc2ed741fbb5c3df9fbcd44a70fe1d208b751a163e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - poetry
+  run:
+    - python
+    - pip
+    - catboost >=0.23
+    - jupyter >=1.0.0
```

Comment:
Could you please double-check this dependencies? I didn't see it on the metadata
```suggestion
```